src/pages/Account/LoginWithCustomToken.tsx

```diff
@@ -15,8 +18,9 @@ import "./Account.scss";
 export interface LoginCustomCodeProps {}
 
 export const LoginWithCustomToken: React.FC<LoginCustomCodeProps> = () => {
-  const { venueId, customToken } = useParams<{
-    venueId?: string;
+  const { worldSlug, spaceSlug, customToken } = useParams<{
+    spaceSlug?: SpaceSlug;
+    worldSlug?: WorldSlug;
     customToken?: string;
   }>();
```

Comment:
[NOTE]
This could be a hook like `useWorldParams` or `useSpaceParams` but instead... `useLoginParams`? It can probably reuse `useSpaceParams` inside it.

Reply:
I'm probably just going to remove this entire component. We don't support it right now anyway.